src/app/accounts/send-balance-to-accounts.ts

```diff
@@ -0,0 +1,25 @@
+import { getRecentlyActiveAccounts } from "@app/accounts/active-accounts"
```

Comment:
not mandatory, but we should try to keep it consistent. Lets discuss tomorrow if we should use this way or relative for imports in the same sub-module